msu/utilities/misc.nut

```diff
@@ -37,3 +37,23 @@
 			throw ::MSU.Exception.InvalidType(_object);
 	}
 }
+
+::MSU.isKindOf <- function( _object, _className )
+{
+	local obj = _object;
+	if (typeof obj == "instance")
+	{
+		if (obj instanceof ::WeakTableRef)
+		{
+			if (obj.isNull())
+			{
+				::printError("The table inside the WeakTableRef instance is null");
+				throw ::MSU.Exception.KeyNotFound(_className);
+			}
```

Comment:
Actually is this even necessary? wouldn't it throw the right error inside of isKindOf even if we didn't do this?